tests/samples/VarDefine.scala

```diff
@@ -0,0 +1,11 @@
+package org.scalanative.bindgen.samples
+
+import scala.scalanative._
+import scala.scalanative.native._
+
+@native.link("bindgentests")
+@native.extern
+object VarDefine {
+  @name("a")
+  val A: native.CInt = native.extern
```

Comment:
When we will support `extern` variables this could be simply:

```scala
def a: native.CInt = extern
def A = a
```

WDYT?

Reply:
Yes, but if a variable is private then it will be filtered, so we should be careful here
```c
extern int __private_int;
#define my_int __private_int
```